cmd/containerd-shim/main_unix.go

```diff
@@ -44,6 +44,10 @@ func main() {
 			Name:  "namespace,n",
 			Usage: "namespace that owns the task",
 		},
+		cli.StringFlag{
+			Name:  "socket,s",
+			Usage: "abstract socket path to server on",
```

Comment:
nit: "...path to serve on" or "...path to run the server on" (or "...path which the server listens on")